pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+markers =
+    gt_3_7: Mark a test as requiring python > 3.7.
```

Comment:
Should we remove python 3.7 support as you suggest?

Reply:
Yes, if we're going to go with that this should go.  I'll remove it here; I can always add it back if the consensus is that we *do* want 3.7.